justfile

```diff
@@ -137,9 +137,14 @@ manage *ARGS: devenv
 
 # run tests
 test *ARGS: devenv
-    $BIN/python -m pytest {{ ARGS }}
+    $BIN/python -m pytest {{ ARGS }}  -k "not functional"
 
 
+# run functional tests
+test-functional *ARGS: devenv
+    $BIN/python -m pytest {{ ARGS }} -k functional
+  
+
```

Comment:
I'm not entirely convinced by these changes, in that I feel like `just test` shouldn't restrict what sort of tests it can run and you can get the equivalent of `just test-functional` with:
```
just test tests/functional
```
Which is admittedly longer, but the path does tab-complete.

But I don't feel particularly strongly about either of these things so quite happy for you to decide.